homeassistant/components/bayesian/strings.json

```
@@ -14,5 +14,192 @@
+            "state": "Add on observation for a sensor's state",
+            "numeric_state": "Add an observation for a numeric range",
+            "template": "Add an observation for a template"
+          }
+        },
+        "state": {
+          "title": "Add a Bayesian sensor",
+          "description": "Add an observation which evalulates to `True` when the value of the sensor exactly matches 'To state'.",
+          "data": {
+            "name": "[%key:common::config_flow::data::name%]",
+            "entity_id": "Entity",
+            "to_state": "To state",
+            "prob_given_true": "Probability when Bayesian sensor `on`",
+            "prob_given_false": "Probability when Bayesian sensor `off`"
+          },
+          "data_description": {
+            "name": "This name will be used for to identify this observation for editing in the future.",
+            "entity_id": "The entity of interest.",
+            "to_state": "The state of the sensor to be detected.",
+            "prob_given_true": "The estimated probability or proportion of time that this observation is `True` when the Bayesian sensor should be 'on'.",
+            "prob_given_false": "The estimated probability or proportion of time that this observation is `True` when the Bayesian sensor should be 'off'."
+          }
+        },
+        "numeric_state": {
+          "title": "[%key:component::bayesian::config_subentries::observation::step::state::title%]",
+          "description": "Add an observation which evaluates to `True` when a numeric sensor is within a chosen range.",
+          "data": {
+            "name": "[%key:common::config_flow::data::name%]",
+            "entity_id": "[%key:component::bayesian::config_subentries::observation::step::state::data::entity_id%]",
+            "above": "Above",
+            "below": "Below",
+            "prob_given_true": "[%key:component::bayesian::config_subentries::observation::step::state::data::prob_given_true%]",
+            "prob_given_false": "[%key:component::bayesian::config_subentries::observation::step::state::data::prob_given_false%]"
+          },
+          "data_description": {
+            "name": "[%key:component::bayesian::config_subentries::observation::step::state::data_description::name%]",
+            "entity_id": "[%key:component::bayesian::config_subentries::observation::step::state::data_description::entity_id%]",
+            "above": "Optional - the lower end of the numeric range. Values exactly matching this will not count",
+            "below": "Optional - the upper end of the numeric range. Values exactly matching this will only count if more than one range is configured for the same entity.",
+            "prob_given_true": "[%key:component::bayesian::config_subentries::observation::step::state::data_description::prob_given_true%]",
+            "prob_given_false": "[%key:component::bayesian::config_subentries::observation::step::state::data_description::prob_given_true%]"
```

Looks like this is a copy & paste error.

```suggestion
            "prob_given_false": "[%key:component::bayesian::config_subentries::observation::step::state::data_description::prob_given_false%]"
```